snap/local/start-mysqld-exporter.sh

```diff
@@ -1,6 +1,23 @@
 #!/bin/bash
 
-# For security measures, daemons should not be run as sudo. Execute mysqlrouter as the non-sudo user: snap-daemon.
-exec $SNAP/usr/bin/setpriv --clear-groups --reuid snap_daemon \
-  --regid snap_daemon -- $SNAP/bin/mysqld_exporter --config.my-cnf=$SNAP_DATA/etc/mysql/mysql.cnf "$@"
+USER="$(snapctl get exporter.user)"
+PASSWORD="$(snapctl get exporter.password)"
+SOCKET="$SNAP_COMMON/var/run/mysqld/mysqld.sock"
```

Comment:
@paulomach  Does this have default behavior if the snap configs are not set?

Reply:
its done